src/patches/ifsc/SBIN0005742.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Data is incorrect in RBI NEFT sheet
+# Points to Delhi instead of Udaipur for CITY
+# Correct data comes from the SBI Branch Locator
+# https://www.sbi.co.in/web/home/locator/branch
+action: patch
+patch:
+  CITY: UDAIPUR
```

Comment:
Correct data doesn't need to be re-patched. Only CENTRE is incorrect, so remainder doesn't need to be patched.